src/components/time-display/time-display.html

```diff
@@ -1,2 +1 @@
-<span>{{time.hours}}<span class="minutes"><span class="timeIndicator">H</span> {{formatNumber(time.minutes)}}<span class="timeIndicator">M</span></span><span class="seconds" *ngIf="time.seconds !== undefined && showSeconds"> {{ formatNumber(time.seconds) }}<span class="timeIndicator">S</span></span>
-</span>
+<span>{{duration.hours()}}<span class="minutes"><span class="timeIndicator">H</span> {{formatNumber(duration.minutes())}}<span class="timeIndicator">M</span></span><span class="seconds" *ngIf="duration.seconds() !== undefined && showSeconds"> {{ formatNumber(duration.seconds()) }}<span class="timeIndicator">S</span></span></span>
```

Comment:
Could you please adjust the formating of this file?

Reply:
Would do.....if I figure out how to proper format this file without screwing up the way it is displayed. Looks like adding line breaks and spaces in span elements causes whitespace to be included.
Going to fix this in my current branch if possible...maybe through something like https://stackoverflow.com/a/5078297